Create ui-c8y-1021-22-114-users-not-able-to-enable-disable-gainsight-px-engagements-kb-bot.md (#3674)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-114-users-not-able-to-enable-disable-gainsight-px-engagements-kb-bot.md

---
date: ""

> title: Fixed issue with saving "Enable in-product information and communication" setting

Reviewer: ```suggestion
title: Fixed issue with saving in-product information setting
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64484
version: 1021.22.114
---

> Previously, changes to the "Enable in-product information and communication" setting were temporary and would reset after reloading the application. This issue has been resolved - the setting now saves permanently and persists after application restart.

Reviewer: ```suggestion
Previously, changes to the "Enable in-product information and communication" setting were temporary and reset after reloading the application. This issue has been resolved. The setting is now saved permanently and persists after application restart.
```

---
date: ""

> title: Users not able to enable/disable Gainsight PX engagements (KB bot) from user settings (#9831) [GRAFT][release/y2025] (#9860)

Reviewer 2: ```suggestion
title: Fixed issue with saving "Enable in-product information and communication" setting
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64484
version: 1021.22.114
---
Co-authored-by: Lukasz Janusz <114157358+luja-c8y@users.noreply.github.com>

> Co-authored-by: Lukasz Janusz <lukasz.janusz@cumuloicty.com>

Reviewer 2: ```suggestion
Previously, changes to the "Enable in-product information and communication" setting were temporary and would reset after reloading the application. This issue has been resolved - the setting now saves permanently and persists after application restart.
```
